libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1148,7 +1148,7 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 		// We could also just use MLOAD; POP right before the gas calculation, but the optimizer
 		// would remove that, so we use MSTORE here.
 		if (!funType.gasSet() && retSize > 0)
-			m_code << "mstore(add(" << fetchFreeMem() << ", " << to_string(retSize) << "), 0)\n";
```

Comment:
I think it's good to keep the mload as part of the string.

Reply:
How would you store to it then?

Reply:
Using a different function.